models/src/cloe/component/osi_sensor.hpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Robert Bosch GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * \file cloe/component/osi_sensor.hpp
+ * \see  cloe/component/osi.hpp
+ */
+
+#pragma once
+#ifndef CLOE_COMPONENT_OSI_SENSOR_HPP_
+#define CLOE_COMPONENT_OSI_SENSOR_HPP_
```

Comment:
We aren't doing these header guards anymore, since no compiler needs them anymore and it's just a mistake we can make.
The pragma once is sufficient.